mender-deb-package

```diff
@@ -149,12 +154,18 @@ build_packages() {
     sign_flags="--sign-key=$key_id"
   fi
 
+  # dh_make needs a name of the user in the env
+  export USER=root
+  # needed to get the orig file
+  echo dh_make --createorig --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
```

Comment:
debug? Please remove the echo - the whole script is executed with `set -x` anyway

Reply:
debug, I will remove.